File: PYTHON/Cap04/c04ex06.py
# c04ex06.py

a = float(input("Entre o 1o. valor numerico: "))
b = float(input("Entre o 2o. valor numerico: "))

print()
print("Escolha uma OPCAO de menu:")
print()
print("[1] - Adicao")
print("[2] - Subtracao")
print("[3] - Multiplicacao")
print("[4] - Divisao")
print()

opcao = int(input("==> "))

print()

if (opcao == 1):
  print("Calculo de adicao")
  print("Resultado:", (a + b))
elif (opcao == 2):
  print("Calculo de subtracao")
  print("Resultado:", (a - b))
elif (opcao == 3):
  print("Calculo de multiplicacao")
  print("Resultado:", (a * b))
elif (opcao == 4):
  print("Calculo de divisao")
  if (b == 0):
    print("Resultado: ERRO")
  else:
    print("Resultado:", (a / b))
else:
  print("Entrada invalida")
  print("Tente novamente em outro momento")

input("\nTecle <Enter> para encerrar... ")

File: KOTLIN/Cap04/c04ex06.kt
// c04ex06.kt

fun main() {
    var A: Float
    var B: Float
    var Opcao: Int

    print("Entre o 1o. valor numerico: ")
    A = readLine()?.toFloat() ?: 0.0f

    print("Entre o 2o. valor numerico: ")
    B = readLine()?.toFloat() ?: 0.0f

    println()
    println("Escolha uma opcao de menu:")
    println()
    println("[1] - Adicao")
    println("[2] - Subtracao")
    println("[3] - Multiplicacao")
    println("[4] - Divisao")
    println()

    print("==> ")
    Opcao = readLine()?.toInt() ?: 0
    println()

    when (Opcao) {
        1 -> {
            println("Calculo de adicao")
            println("Resultado: %.2f".format(A + B))
        }
        2 -> {
            println("Calculo de subtracao")
            println("Resultado: %.2f".format(A - B))
        }
        3 -> {
            println("Calculo de multiplicacao")
            println("Resultado: %.2f".format(A * B))
        }
        4 -> {
            println("Calculo de divisao")
            if (B == 0.0f) {
                println("Resultado: ERRO")
            } else {
                println("Resultado: %.2f".format(A / B))
            }
        }
        else -> {
            println("Entrada invalida")
            println("Tente novamente em outro momento")
        }
    }

    print("\nTecle <Enter> para encerrar... ")
    readLine()
}
